apps/web/src/server/applications/case/s51/applications-s51.controller.js

```diff
@@ -192,6 +192,10 @@ export async function changeAdviceStatus(request, response) {
 	if (body?.isRedacted &&  body.isRedacted === '1') {
 		redacted = true;
 	}
+	// @ts-ignore
```

Comment:
we could have done both ifs with a 1-liner, same as how documents does it:

const redacted = typeof isRedacted === 'string' ? { redacted: isRedacted === '1' } : {};

This is a comment only though, not an approval failure